docs/source/how_to_guides/cloud_providers.md

```diff
@@ -0,0 +1,80 @@
+# Cloud provider
+
+Streaming dataset supports the following cloud storage providers: [AWS S3](https://aws.amazon.com/s3/), [GCP Storage](https://cloud.google.com/storage), and [OCI Cloud Storage](https://www.oracle.com/cloud/storage/) to stream your data directly to your instance. For [MosaicML platform](https://www.mosaicml.com/cloud) users, follow the steps mentioned in this [AWS S3](https://mcli.docs.mosaicml.com/en/latest/secrets/s3.html), [GCP Storage](https://mcli.docs.mosaicml.com/en/latest/secrets/gcp.html), and [OCI Cloud Storage](https://mcli.docs.mosaicml.com/en/latest/secrets/oci.html) doc on how to configure the cloud provider credentials.
+
+## AWS S3
+
+For S3 bucket with a public access, skip rest of the below steps since no credentials setup is required.
```

Comment:
Just get to the point...
```suggestion
For S3 bucket with a public access, no additional setup is required, simply specify the S3 URI of the resource.
```

Reply:
Done in https://github.com/mosaicml/streaming/pull/169/commits/018e7db3bb1bd25ef935e5916d831307b33033f0.